nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1759,6 +1759,11 @@ pub struct NetworkInterfacePathParam {
 }
 
 /// Detach a network interface from an instance.
+///
+/// Note that the primary interface for an instance cannot be deleted if there
+/// are any secondary interfaces. A new primary interface must be designated
+/// first. The primary interface can be deleted if there are no secondary
```

Comment:
Is it actually possible to designate a new primary interface yet? This wording "A new primary interface must be designated first" implies there is a mechanism to switch primaries without deleting all interfaces.

Looking at the implementation of the queries, this doesn't seem possible - which is fine - but the docs here confused me a bit.

Reply:
No, it's not, apologies about that. I'll add a TODO and issue to track updating a NIC on an instance.

Reply:
https://github.com/oxidecomputer/omicron/issues/1153